### DISCUSSION TITLE
How to recover deleted record in PostgreSql database?

### DISCUSSION THREAD

**Discussion by mateusbmsfc** [dbeaver/dbeaver#13014] [General]
Fiz a exclusão de uma linha de registro, tem como recuperar essa linha? 

_I deleted the input line, is there any way to restore this line?_

**Comment by uslss** [comment]
probably, you'll find some info here https://stackoverflow.com/questions/20113586/restore-deleted-records-in-postgresql

please, write in English in future :)
